uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -11,14 +11,14 @@ <h4 class="modal-title">
 </div>
 <crm-loading show="!$ctrl.loading.absenceTypes">
   <div
-    ng-show="$ctrl.isRole('manager')"
+    ng-show="$ctrl.isRole('manager') && !$ctrl.isMode('create')"
```

Comment:
This is the block which displays the users Name and Status of the Request, and it should only be visible when It a manager and he is reviewing an old request. Previously the `$ctrl.isRole('manager')` was only true when the user is manager but not in create mode. As the `$ctrl.isRole('manager')` is fixed now, the `!$ctrl.isMode('create')` is necessary to be added.